workbench/src/renderer/components/SettingsModal/index.jsx

```diff
@@ -164,6 +164,23 @@ class SettingsModal extends React.Component {
                 </Form.Control>
               </Col>
             </Form.Group>
+            <Form.Group as={Row}>
+              <Form.Label column sm="6" htmlFor="gdal-logging-select">
+                {t('GDAL logging level')}
+              </Form.Label>
+              <Col sm="6">
+                <Form.Control
+                  id="gdal-logging-select"
+                  as="select"
+                  name="gdalLoggingLevel"
+                  value={investSettings.gdalLoggingLevel}
+                  onChange={this.handleChange}
+                >
+                  <option value="NORMAL" key="NORMAL">NORMAL</option>
+                  <option value="DEBUG" key="DEBUG">DEBUG</option>
```

Comment:
We should wrap the displayed strings to get translated `<...>{t('NORMAL')}</...>`

Reply:
patched in bd9a9b64c